src/com/scottyplunkett/server/Combiner.java

```diff
@@ -0,0 +1,11 @@
+package com.scottyplunkett.server;
+
+class Combiner {
```

Comment:
😬

Reply:
Maybe something more specific? This name does not speak to what the class is combining.